Scripts/Python/plasma/Plasma.py

```diff
@@ -4271,6 +4271,10 @@ def getKey(self):
         """Returns the ptKey for this GUI control"""
         pass
 
+    def getMargins(self) -> Tuple[int, int, int, int]:
+        """Returns a tuple of (top, left, right, bottom) margins"""
```

Comment:
This comment doesn't match the ordering in `setMargins`: (top, left, bottom, right). I think the order in `setMargins` is or probably should be the correct one, although if your intention was to match something like CSS's margin/position descriptor rules, it'd be (top, right, bottom, left), IIRC.

Reply:
Plasma's convention here matches the Win32 coordinate system.

Reply:
Understandable, I've also seen the Win32 coordinate system as well. I just wanted to make sure I understood your intention. In any case, the comments should match the reality, though. ;)

Reply:
Agreed. I'm going through and fixing things now.